blazor/rich-text-editor/iframe.md

```diff
@@ -44,9 +44,9 @@ The editor allows you to pass an additional attribute to body tag of a `<iframe>
 
 ## Adding external CSS/Script file
 
-The editor offers to add external CSS file to style the `<iframe>` element. Easily change the appearance of editor’s content using an external CSS file using `Resources - Styles` field in the `RichTextEditorIframeSettings` property.
+The editor offers to add external CSS file to style the `<iframe>` element. Easily change the appearance of editor’s content using an external CSS file using **Resources - Styles** field in the `RichTextEditorIframeSettings` property.
```

Comment:
Link the API class reference as a complex type like - https://blazor.syncfusion.com/documentation/rich-text-editor/iframe#iframe-attributes

Reply:
API link provided